src/components/BounceRuleDetailed/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
```

Comment:
We can change the naming convention of the component to be BounceRuleDetails rather than Detailed past-tense. also extra .DS_store file again. you can add that to your .gitignore instead so it won't add it on commits